_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -90,34 +90,32 @@ You will **not** need to use your **source 3DS** to complete any further steps o
 
 #### Section IV - unSAFE_MODE
 
-If your (Right/Left Shoulder), (D-Pad Up), or (A) buttons do not work, you will not be able to follow the instructions. For assistance with this matter, join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask, in English, for help.
-{: .notice--warning}
-
-1. With the target 3DS still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), then press (Power)
-  + Keep holding the buttons until the target 3DS boots into Safe Mode
-  + Safe Mode will appear as a System Update screen
+1. With your device still powered off, hold the following buttons: (Left Shoulder) + (Right Shoulder) + (D-Pad Up) + (A), and while holding these buttons together, power on your device
```

Comment:
device > target 3DS

Reply:
is that really necessary when the guide says that the source 3ds wont be needed for any further steps

Reply:
~~do people read?~~

I guess you have a point. Let's just leave it as is for now, then.